scripts/bm.rb

```diff
@@ -0,0 +1,43 @@
+# Runs a mild benchmark and prints out the average time a call to 'throttle!' takes.
+
+require 'prorate'
+require 'benchmark'
+require 'redis'
+require 'securerandom'
+
+def average_ms(ary)
+  ary.map{|x| x*1000}.inject(0,&:+) / ary.length
+end
+
+r = Redis.new
+
+logz = Logger.new(STDERR)
+logz.level = Logger::FATAL # block out most stuff
+
+times = []
+15.times do
+  times << Benchmark.realtime {
+    t = Prorate::Throttle.new(redis: r, logger: logz, name: "throttle-login-email", limit: 60, period: 120, block_for: 5)
+    # Add all the parameters that function as a discriminator
+    t << '127.0.2.1'
+    t << 'onno@wetransfer.com'
```

Comment:
Changed to something else. It'll still be there for people digging through the history though :(